### PR TITLE
pop up info for a single event

### PR DESCRIPTION
**Pull request by nicolascr181**

### CODE REVIEW

**Comment by nicolascr181** [comment]
Captura

**Comment by nicolascr181** [comment]
image

File: dashboard/src/components/Directory/Map.vue [hunk 1]
```diff
@@ -1,30 +1,215 @@
 <template>
   <l-map style="height: 100%" :zoom="zoom" :center="center">
     <l-tile-layer :url="url" :attribution="attribution"></l-tile-layer>
-    <l-marker :lat-lng="markerLatLng"></l-marker>
+    <l-marker :lat-lng="markerLatLng" @click="getEvent">
```
Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue]: with this behavior we need have a modal for each marker, I suggest a method that get data, fill the pop up, and show the popo up, and the pop up can be a component

**Review by 1701722767 — CHANGES_REQUESTED** [review]

**Comment by nicolascr181** [comment]
image

File: dashboard/src/components/Directory/Container.vue [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default {
 <style scoped>
 .container-directory {
   height: 100%;
-  padding-bottom: 56px !important;
+
```
Comment (1701722767) [inline dashboard/src/components/Directory/Container.vue]: delete space

File: dashboard/src/components/Directory/Map.vue [hunk 3]
```diff
@@ -1,30 +1,74 @@
...
+      },
+
     };
-  }
-}
+  },
+
+  methods : {
+      showPopUp (){
+        this.$refs.PopUp.show('/find', this.params);
```
Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue]: the params can be a param in the method

Comment (1701722767) [inline dashboard/src/components/Directory/Map.vue]: /events/find not?

**Review by 1701722767 — CHANGES_REQUESTED** [review]

**Review by 1701722767 — APPROVED** [review]

**Review by sergiomonz — APPROVED** [review]

**Pull request merged**